app/controllers/application_controller.rb

```diff
@@ -11,8 +11,6 @@ class ApplicationController < ActionController::Base
 
   default_form_builder GOVUKDesignSystemFormBuilder::FormBuilder
 
-  before_action :clear_draft_questions_data
```

Comment:
I did work on this a bit with @aliuk2012 , this may have been here as a legacy from a time when incomplete questions were stored in the session, so we were a lot more concerned about storing an indefinite amount of stuff.